common/djangoapps/student/models.py

```diff
@@ -1355,6 +1355,7 @@ def update_enrollment(self, mode=None, is_active=None, skip_refund=False):
         if activation_changed:
             if self.is_active:
                 self.emit_event(EVENT_NAME_ENROLLMENT_ACTIVATED)
+                self.send_signal(EVENT_NAME_ENROLLMENT_ACTIVATED)
```

Comment:
@viadanna, did you mean `EnrollStatusChange.enroll`?

Reply:
@Agrendalath This is the same event name stored in tracking logs, so I think it's more explicit.

Reply:
@viadanna, we are sending ["edx.course.enrollment.activated"](https://github.com/edx/edx-platform/blob/484cd536e25ca74b4354c6b61c52084d8b935bf2/common/djangoapps/student/models.py#L992) with this.
The `enroll` signal is sending ["enroll"](https://github.com/edx/edx-platform/blob/484cd536e25ca74b4354c6b61c52084d8b935bf2/common/djangoapps/student/models.py#L111). 

These events are emitted by the trackers, which are used, e.g for the tracking logs. We should use Django signals here.

Reply:
@Agrendalath Oh, that makes absolute sense. The enrollment deactivation sends unenroll as well.